app/templates/components/uk_components/footer/template.njk

```diff
@@ -61,14 +61,14 @@
 
 
 <footer class="usa-footer usa-footer--big">
-  <div class="grid-container usa-footer__return-to-top">
+  {# <div class="grid-container usa-footer__return-to-top">
     <a href="#">Return to top</a>
-  </div>
+  </div> #}
```

Comment:
Using Jinja comments on plain HTML seems like a good idea, but I also think we should be consistent in comment styling to help our future selves. Let's have this either be an HTML comment or switch the other HTML comments to Jinja comments.